packages/language-tools/test/generate-esjs-syntax.test.js

```
@@ -0,0 +1,39 @@
+import { describe, it, expect } from 'vitest';
+import fs from 'fs';
```

Biome

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)